### PR TITLE
key notatinon changed fix Lp1813436

### PR DESCRIPTION
**Pull request by daschuer**
This PR Fixes https://bugs.launchpad.net/mixxx/+bug/1813436 and other relates issue around propagating the selected key notation inside Mixxx. Now "[Library]", "key_notation" is used everywhere to signal key notation changes.

### CODE REVIEW

File: src/widget/wkey.cpp [hunk 1]
```diff
@@ -56,8 +56,6 @@ void WKey::setCents() {
     setValue(m_dOldValue);
 }
 
-void WKey::preferencesUpdated(double dValue) {
-    if (dValue > 0) {
-        setValue(m_dOldValue);
-    }
+void WKey::keyNotationChanged(double dValue) {
+    setValue(m_dOldValue);
```
Comment (uklotzde) [inline src/widget/wkey.cpp]: warning: unused parameter ‘dValue’ [-Wunused-parameter]

Comment (daschuer) [reply src/widget/wkey.cpp]: Ups, ... yes. this value should be used. Fixed.

**Comment by daschuer** [comment]
Again read for merge.

**Comment by uklotzde** [comment]
LGTM

**Comment by uklotzde** [comment]
@daschuer I didn't notice that Mixxx is churning CPU like hell, from < 10-15% to now 50% when idling.

A git bisect revealed this commit as the cause:
```
9640a6f0ec3231c47652dfa39bfcb5f88c24bd11 is the first bad commit
commit 9640a6f0ec3231c47652dfa39bfcb5f88c24bd11
Author: Daniel Schürmann <daschuer@mixxx.org>
Date:   Sat Mar 23 23:01:08 2019 +0100

    Update TrackTableview after key notation changed

:040000 040000 8baced4a714cf17970222eb40d6f237df06a0a14 c374550ab3565e735b4454070c3608958a5700cf M	src
```

**Comment by daschuer** [comment]
Ups ....
This is https://github.com/mixxxdj/mixxx/commit/9640a6f0ec3231c47652dfa39bfcb5f88c24bd11
A widget update after [Library]","key_notation" has changed.

**Pull request merged**